lib/raw-stream-set.js

```diff
@@ -26,4 +26,8 @@ module.exports = class RawStreamSet {
       self._streams.delete(id)
     }
   }
+
+  async destroy () {
+    for (const stream of this._streams.values()) stream.destroy()
```

Comment:
Not sure if this should maybe wait for the streams to emit `close`?

Reply:
I think it should

Reply:
@mafintosh Should be all set now.